infra/bridge/src/controllers/promotions.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+const { getAsync } = require('../utils/redis')
+
+const Attestation = require('./../models/attestation')
+
+// TODO: Add request body validation
+router.post('/verify', async (req, res) => {
+  const { type, socialNetwork, identity } = req.body
+
+  const attestation = await Attestation.findOne({
+    ethAddress: identity,
+    method: socialNetwork
+  })
+
+  if (!attestation) {
+    return res.status(400).send({
+      success: false,
+      errors: [`Attestation missing`]
+    })
+  }
+
```

Comment:
nit: should we have some defensive check to make sure type of either SHARE or FOLLOW ?

Reply:
We can do that in request body validation. Will do that.